modules/apps/object/object-service/src/test/java/com/liferay/object/internal/search/spi/model/permission/ObjectEntrySearchPermissionFilterContributorTest.java

```diff
@@ -0,0 +1,215 @@
+import com.liferay.portal.test.rule.LiferayUnitTestRule;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+
+import org.mockito.Mockito;
+
+/**
+ * @author Gustavo Lima
+ */
+public class ObjectEntrySearchPermissionFilterContributorTest {
+
+	@ClassRule
+	@Rule
+	public static final LiferayUnitTestRule liferayUnitTestRule =
+		LiferayUnitTestRule.INSTANCE;
+
+	@Test
+	public void testContribute() throws PortalException {
+		long activeAccountEntryId = RandomTestUtil.randomLong();
+		long inactiveAccountEntryId = RandomTestUtil.randomLong();
+		long userId = RandomTestUtil.randomLong();
+
+		ObjectEntrySearchPermissionFilterContributor
+			objectEntrySearchPermissionFilterContributor =
+				_setUpObjectEntrySearchPermissionFilterContributor(
+					activeAccountEntryId, inactiveAccountEntryId, userId);
+
+		BooleanFilter booleanFilter = new BooleanFilter();
+
+		objectEntrySearchPermissionFilterContributor.contribute(
+			booleanFilter, RandomTestUtil.randomLong(), new long[0], userId,
+			_getPermissionChecker(userId), ObjectDefinition.class.getName());
+
+		String booleanFilterString = booleanFilter.toString();
+
+		Assert.assertFalse(
+			booleanFilterString,
+			booleanFilterString.contains(
+				"accountEntryRestrictedObjectFieldValue=[" +
+					inactiveAccountEntryId + "]"));
+		Assert.assertTrue(
+			booleanFilterString,
+			booleanFilterString.contains(
+				"accountEntryRestrictedObjectFieldValue=[" +
+					activeAccountEntryId + "]"));
+	}
+
+	private AccountEntryUserRel _createAccountEntryRel(
+			long accountEntryId, boolean inactive)
+		throws PortalException {
+
+		AccountEntryUserRel accountEntryUserRel = Mockito.mock(
+			AccountEntryUserRel.class);
+
+		AccountEntry accountEntry = Mockito.mock(AccountEntry.class);
+
+		Mockito.when(
+			accountEntryUserRel.getAccountEntry()
+		).thenReturn(
+			accountEntry
+		);
+
+		Mockito.when(
+			accountEntry.getAccountEntryId()
+		).thenReturn(
+			accountEntryId
+		);
+
+		Mockito.when(
+			accountEntry.isInactive()
+		).thenReturn(
+			inactive
+		);
+
+		return accountEntryUserRel;
+	}
+
+	private PermissionChecker _getPermissionChecker(long userId) {
+		PermissionChecker permissionChecker = Mockito.mock(
+			PermissionChecker.class);
+
+		Mockito.when(
+			permissionChecker.getUserId()
+		).thenReturn(
+			userId
+		);
+
+		return permissionChecker;
+	}
+
+	private void _setUpAccountEntryRels(
+			long activeAccountEntryId, long inactiveAccountEntryId,
+			ObjectEntrySearchPermissionFilterContributor
+				objectEntrySearchPermissionFilterContributor,
+			long userId)
+		throws PortalException {
+
+		AccountEntryUserRelLocalService accountEntryUserRelLocalService =
+			_setUpAccountEntryUserRelLocalService(
+				objectEntrySearchPermissionFilterContributor);
+
+		List<AccountEntryUserRel> accountEntryRels = Arrays.asList(
```

Comment:
Can you rename it to `AccountEntryUserRels`? ( I think you miss the AccountEntry**User**Rels)